src/Tizen.Sensor/Tizen.Sensor/Plugins/AutoRotationSensor.cs

```diff
@@ -71,10 +71,6 @@ public static bool IsSupported
         /// </param>
         public AutoRotationSensor(uint index = 0) : base(index)
         {
-            if (!IsSupported)
-            {
-                throw new NotSupportedException("Not Supported: " + "AutoRotationSensor is not supported");
-            }
             Log.Info(Globals.LogTag, "Creating AutoRotationSensor object");
```

Comment:
any reason of why remove this check?

Reply:
Because I thought a problem would occur if I throw an exception here.. I think it was my misunderstanding.. It's removed now.